index.html

```diff
@@ -3,11 +3,21 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>有限会社ヤタカ物産　建材各種販売</title>
     <link href="/css/reset.css"  rel="stylesheet" />
     <link href="/css/style.css"  rel="stylesheet" />
 </head>
 <body>
-    <h1>Hello World</h1>
+    <header class="site-header">
+        <div class="header-left">
+            <img src="/img/logo.png" alt="ヤタカ物産ロゴ" class="logo">
+        </div>
+        <div class="header-right">
+            <a href="@" class="contact-link">
```

Comment:
ダミーリンクは ただの空文字でOKです！（かつては `#` だったけど今はこれが正しいHTMLの仕様）
```suggestion
            <a href="" class="contact-link">
```